modules/nw-networkpolicy-optimize-ovn.adoc

```diff
@@ -3,11 +3,11 @@
 // * networking/network_policy/about-network-policy.adoc
 
 [id="nw-networkpolicy-optimize-ovn_{context}"]
-= Optimizations for network policy with OpenShift OVN
+= Optimizations for network policy with OVN-Kubernetes network plugin
 
 When designing your network policy, refer to the following guidelines:
 
-* For network policies with the same `spec.podSelector` spec,  it is more efficient to use one network policy with multiple `ingress` or `egress` rules, than multiple network policies with subsets of `ingress` or `egress` rules.
+* For network policies with the same `spec.podSelector` spec, it is more efficient to use one network policy with multiple `ingress` or `egress` rules, than multiple network policies with subsets of `ingress` or `egress` rules.
```

Comment:
@jab-rh if in this example 1, the point being put is multiple policies versus single policy, then make test-network-policy as test-network-policy1 and  test-network-policy2.
policy 1
<pre>
apiVersion: networking.k8s.io/v1
kind: NetworkPolicy
metadata:
  name: test-network-policy1
spec:
  podSelector: {}
  ingress:
  - from:
    - podSelector:
        matchLabels:
          role: frontend
</pre>
policy 2
<pre>
apiVersion: networking.k8s.io/v1
kind: NetworkPolicy
metadata:
  name: test-network-policy2
spec:
  podSelector: {}
  ingress:
  - from:
    - podSelector:
        matchLabels:
          role: backend
</pre>

Reply:
@asood-rh, I think the intent here is to show that one policy with multiple `ingress.from` items is worse than consolidating them together with `matchExpressions` if it is possible to do so.